src/CMakeLists.txt

```
@@ -86,11 +86,19 @@ endif()
 # link libraries and header which are then excluded from runtime component
 set (CMAKE_INSTALL_DEFAULT_COMPONENT_NAME "xrt")
 
-# Enable development package by specifying development component name
+# Enable development package by setting XRT_DEV_COMPONENT to "xrt-dev"
 # If XRT_DEV_COMPONENT is same DEFAULT_COMPONENT then only that package
 # is created with both development and run-time content.
 #set (XRT_DEV_COMPONENT "xrt-dev")
-set (XRT_DEV_COMPONENT "xrt")
+option (XRT_DEV_COMPONENT "Set to 'xrt-dev' to split dev artifacts into separate component&package" "xrt")
```

CMake options are used to set things ON or OFF, I could not get this working without converting the option to a cached CMake variable.  
```suggestion
set (XRT_DEV_COMPONENT "xrt" CACHE STRING "Set to 'xrt-dev' to split dev artifacts into separate component&package")
```